ShowContact/index.ts

```diff
@@ -1,31 +1,45 @@
 import { AzureFunction, Context, HttpRequest } from "@azure/functions";
 import { respondWithSuccess } from "../lib/responders/Responses";
+import { contactsRepository } from "../repository/contactsRepository";
+import { Contact } from "../models/Contact";
+import _ from "lodash";
+
 const httpTrigger: AzureFunction = async function (
   context: Context,
   req: HttpRequest
 ): Promise<void> {
-  const contact = context.bindings.contact;
+  const contactDao = new contactsRepository();
+  await contactDao.init();
+
+  const contact  = await contactDao.findContactById(
+    context.bindingData.id,
+    context.bindingData.clientId
+  );
   if (!contact) {
     context.res = {
       status: 404,
       body: "Document with id " + context.bindingData.id + " not found",
     };
   } else {
+   const contactResponse =_.pick(contact, [
+      "id",
+      "mobileNumber",
+      "firstName",
+      "lastName",
+      "emailAddress",
+      "city",
+      "country",
+      "dateOfBirth",
+      "clientId",
+    ])
     respondWithSuccess(context, {
       type: "contact",
       id: contact.id,
       clientId: context.bindingData.clientId,
       attributes: {
-        mobileNumber: contact.mobileNumber,
-        firstName: contact.firstName,
-        lastName: contact.lastName,
-        emailAddress: contact.emailAddress,
-        city: contact.city,
-        country: contact.country,
-        dateOfBirth: contact.dateOfBirth,
+        data:contactResponse
```

Comment:
There should not be a `data` sub property here. Take a look at object destructuring using rest command in javascript